lib/ansible/modules/cloud/amazon/lightsail.py

```diff
@@ -323,8 +318,7 @@ def main():
         wait_timeout=dict(default=300, type='int'),
     )
 
-    module = AnsibleAWSModule(argument_spec=argument_spec,
-                              required_if=[['state', 'present', ('zone', 'blueprint_id', 'bundle_id', 'key_pair_name')]])
+    module = AnsibleAWSModule(argument_spec=argument_spec)
```

Comment:
Are zone, blueprint_id and bundle_id not required any more?

Reply:
Sorry, I guess I was too focused on the key_pair_name option. This required_if is still needed. Will add it in.